core_ui/src/main/java/com/alexafit/core_ui/Typography.kt

```diff
@@ -0,0 +1,97 @@
+package com.alexafit.core_ui
+
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+data class Inter(
+    val bold: FontFamily =
+        FontFamily(
+            Font(R.font.inter_regular, FontWeight.Normal)
+        ),
+    val semiBold: FontFamily =
+        FontFamily(
+            Font(R.font.inter_semibold, FontWeight.SemiBold)
+        ),
+    val regular: FontFamily =
+        FontFamily(
+            Font(R.font.inter_regular, FontWeight.Normal)
+        ),
+    val light: FontFamily =
+        FontFamily(
+            Font(R.font.inter_light, FontWeight.Light)
+        )
+)
```

Comment:
Could this be an `object` so you don't always need to create an instance when referring to font families?